docs/troubleshoot-collection.md

```diff
@@ -405,6 +471,32 @@ The duplicated pod deletion command is there to make sure the pod is not stuck i
 If you observe that Prometheus Pod needs more and more resources (out of memory failures - OOM killed Prometheus) and you are not able to
 increase them then you may need to horizontally scale Prometheus. :construction: Add link to Prometheus sharding doc here.
 
+### Otelcol enqueue failures
+
+Enqueue failures happen when otelcol can't write to its persistent queue. They signify data loss, as otelcol is unable to locally buffer the
+data, and is forced to drop it.
+
+#### Queue full
+
+Most of the time, this happens because the queue is full, which is almost always caused by otelcol not being able to send data to Sumo, and
+should be accompanied by other alerts. Resolving the underlying issue will resolve the alert - in this case, this alert is informational.
+
+The queue size can be checked on the Otelcol dashboard, available in [the collection dashboards folder](#collection-dashboards).
+
```

Comment:
Shouldn't we have separate section or document for alerts? We do not have such right now, so it can be not clear for customers what they are and how do the work

Reply:
Maybe we leave alerts out of this document and instead here I could add here the query and a screenshot for a full queue? (for the customer's reference)

Reply:
Yeah, it sounds reasonable